From abf45bf51661cc47f82e12c9aedbfb962cd43e3e Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Mon, 20 Sep 2021 01:34:17 +0300
Subject: [PATCH 1/1] Revert "info-overview: Implements usage of os-release to
 search for the OS logo"

This reverts commit f39cd5a93b0dcddda9911586820ed55bbbe2eb3d.
---
 panels/info-overview/cc-info-overview-panel.c  | 12 ------------
 panels/info-overview/cc-info-overview-panel.ui |  6 ++++--
 2 files changed, 4 insertions(+), 14 deletions(-)

diff --git a/panels/info-overview/cc-info-overview-panel.c b/panels/info-overview/cc-info-overview-panel.c
index 78fba73a6..deb84ed49 100644
--- a/panels/info-overview/cc-info-overview-panel.c
+++ b/panels/info-overview/cc-info-overview-panel.c
@@ -68,7 +68,6 @@ struct _CcInfoOverviewPanel
   CcListRow       *hostname_row;
   CcListRow       *memory_row;
   GtkListBox      *os_box;
-  GtkImage        *os_logo;
   CcListRow       *os_name_row;
   CcListRow       *os_type_row;
   CcListRow       *processor_row;
@@ -876,14 +875,6 @@ use_dark_theme (CcInfoOverviewPanel *panel)
   return (theme_name != NULL && g_str_has_suffix (theme_name, "dark")) ? TRUE : FALSE;
 }
 
-static void
-setup_os_logo (CcInfoOverviewPanel *panel)
-{
-  g_autofree char *logo_name = g_get_os_info ("LOGO");
-  gtk_image_set_from_resource (panel->os_logo, "/org/gnome/control-center/info-overview/GnomeLogoVerticalMedium.svg");
-  gtk_image_set_pixel_size (panel->os_logo, 256);
-}
-
 static void
 cc_info_overview_panel_class_init (CcInfoOverviewPanelClass *klass)
 {
@@ -902,7 +893,6 @@ cc_info_overview_panel_class_init (CcInfoOverviewPanelClass *klass)
   gtk_widget_class_bind_template_child (widget_class, CcInfoOverviewPanel, hostname_row);
   gtk_widget_class_bind_template_child (widget_class, CcInfoOverviewPanel, memory_row);
   gtk_widget_class_bind_template_child (widget_class, CcInfoOverviewPanel, os_box);
-  gtk_widget_class_bind_template_child (widget_class, CcInfoOverviewPanel, os_logo);
   gtk_widget_class_bind_template_child (widget_class, CcInfoOverviewPanel, os_name_row);
   gtk_widget_class_bind_template_child (widget_class, CcInfoOverviewPanel, os_type_row);
   gtk_widget_class_bind_template_child (widget_class, CcInfoOverviewPanel, processor_row);
@@ -930,8 +920,6 @@ cc_info_overview_panel_init (CcInfoOverviewPanel *self)
 
   info_overview_panel_setup_overview (self);
   info_overview_panel_setup_virt (self);
-
-  setup_os_logo (self);
 }
 
 GtkWidget *
diff --git a/panels/info-overview/cc-info-overview-panel.ui b/panels/info-overview/cc-info-overview-panel.ui
index 95afe867c..0926962e0 100644
--- a/panels/info-overview/cc-info-overview-panel.ui
+++ b/panels/info-overview/cc-info-overview-panel.ui
@@ -25,10 +25,12 @@
                 <property name="spacing">30</property>
                 <property name="orientation">vertical</property>
                 <child>
-                  <object class="GtkImage" id="os_logo">
+                  <object class="GtkImage">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
-                    <property name="use_fallback">True</property>
+                    <property name="pixel_size">192</property>
+                    <property name="icon_name">start-here-solus</property>
+                    <property name="icon_size">6</property>
                   </object>
                   <packing>
                     <property name="expand">False</property>
-- 
2.32.0

